File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        String str = s.toLowerCase();
        int pLen = str.length() - str.replace("p", "").length();
		int yLen = str.length() - str.replace("y", "").length();
        
        boolean answer = pLen == yLen ? true : false;
    
        return answer;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;
    var pCnt = 0;
    var yCnt = 0;
    s = s.toLowerCase();
    for (let i in s) {
        if (s[i] === "p") pCnt++;
        else if (s[i] === "y") yCnt++;
    }

    return pCnt === yCnt;
}